### ISSUE TITLE
Free shipping threshold is mistakenly synced as minimum spending threshold

### ISSUE THREAD

**Issue by eason9487** [woocommerce/google-listings-and-ads#2469]
### Describe the bug:

If there is a minimum order amount to get free shipping, it will be mistakenly synced to Google Merchant Center as a minimum spending threshold.

This issue occurs with shipping data obtained from WooCommerce core or set in this extension.

#### Using WooCommerce core shipping data

1. Go to **WooCommerce > Settings > Shipping > Shipping zones** and set up a zone with free shipping and minimum order amount.
   ![image](https://github.com/user-attachments/assets/e6537348-2d21-4f27-b4ef-6eb69419505c)
2. Go to edit free listings.
3. Select the **Recommended:  Automatically sync my store’s shipping settings to Google** option for the shipping rates setting.
4. Add the above configured zone to the Audience setting.
5. Save changes.
6. Wait a moment for the settings to sync to Google Merchant Center.
7. Go to Google Merchant Center and navigate to **Shipping and returns** page.
8. Expand the corresponding shipping policy.
   - The above minimum order amount of free shipping is set as the **No shipping under** amount.
   - The **Free shipping over** amount is empty

![1](https://github.com/user-attachments/assets/8aaa52d1-24f4-4e48-ba08-3e64f47fde11)

#### Using extension shipping data

1. Go to edit free listings.
2. Select the **My shipping settings are simple. I can manually estimate flat shipping rates** option for the shipping rates setting.
3. Add a free shipping threshold.
   ![image](https://github.com/user-attachments/assets/39614f28-ddc6-45d5-8eeb-6a0df17247bb)
4. Save changes.
5. Go to Google Merchant Center and navigate to **Shipping and returns** page to see the incorrect synchronization.

![1](https://github.com/user-attachments/assets/1bc15e4a-90de-4c34-bc86-06c9778dfedd)

### Expected behavior:

The minimum order amount to get free shipping should be synced to Google Merchant Center as the **Free shipping over** amount.

### Actual behavior:

The minimum order amount to get free shipping is synced to Google Merchant Center as the **No shipping under** amount.

#### Related codes:

- https://github.com/woocommerce/google-listings-and-ads/blob/2.7.6/src/Shipping/GoogleAdapter/WCShippingSettingsAdapter.php#L167-L173
- https://github.com/woocommerce/google-listings-and-ads/blob/2.7.6/src/Shipping/GoogleAdapter/DBShippingSettingsAdapter.php#L79-L90

#### The `minimumOrderValue` field in Content API for Shopping document: 

https://developers.google.com/shopping-content/reference/rest/v2.1/shippingsettings#service

> Minimum order value for this service. If set, indicates that customers will have to spend at least this amount.

**Comment by tomalec** [comment]
I'd like to question the severity of this issue. As if I understand correctly, we indeed use the `x` free shipping threshold as the  "No shipping under" instead of "Free shipping over," but we do so for the shipping group with cost == 0. So logically, it means zero-cost for orders over x.

So, instead of, creating one shipping group:
- "Shipping rate 99.00, free shipping for orders over 987.00"
We create two
- "Shipping rate 0, for orders over 987"
- "Shipping rate 99.00, for all"

So, both, IMHO, describe more-or-less the same thing. I agree that they are not precisely the same, and it would be better for the merchant experience to create n/2 groups and use that field.

Another thing is that, in API docs, I didn't find a way to specify a free shipping threshold :/

**Comment by puntope** [comment]
I agree with @tomalec  here. Google Service does create a single group for each threshold that is equivalent to the expected behaviour. So I'm setting this as low for now. Feel free @eason9487 to revert this if I'm missing something about the issue that makes it a High Prio Bug.

**Comment by puntope** [comment]
Tested this a bit more... When I enter in MC and edit the Shipping I cannot set Excluded Value as 0 anymore 😕 

<img width="1231" alt="Image" src="https://github.com/user-attachments/assets/46f7c3f2-27cf-44f5-b0f5-5102658d17c0" />

**Comment by eason9487** [comment]
Thank you all for further investigation! Agree that the severity should have been set as low.

> Another thing is that, in API docs, I didn't find a way to specify a free shipping threshold

Looks like there are free ship rate examples on this guide page: https://developers.google.com/shopping-content/guides/shippingsettings#shippingsettingsupdate. Maybe it's something to try out.

**Comment by SpeedyBroz** [comment]
Suggest increasing severity due to Google Marketplace using this bug as a reason to repeatedly suspend merchant account due to "misrepresentation"
